src/async/async_api.c

```diff
@@ -240,7 +240,7 @@ __async_start(WT_SESSION_IMPL *session)
 	async = conn->async;
 	TAILQ_INIT(&async->formatqh);
 	WT_RET(__wt_spin_init(session, &async->ops_lock, "ops"));
-	WT_RET(__wt_cond_alloc(session, "async flush", &async->flush_cond));
+	WT_RET(__wt_cond_init(session, &async->flush_cond, "async flush"));
```

Comment:
Was there a reason to swap the order of the args?  I don't have strong feelings either way, it just seems gratuitous.